Frameworks/UIKit/WYPopoverController.m

```diff
@@ -33,23 +33,27 @@ of this software and associated documentation files (the "Software"), to deal
     #error ARC must be enabled
 #endif
 
-//#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 70000
+#if defined(WINOBJC) || __IPHONE_OS_VERSION_MAX_ALLOWED >= 70000
 #define WY_BASE_SDK_7_ENABLED
-//#endif
+#endif
 
 #ifdef DEBUG
 #define WY_LOG(fmt, ...)		NSLog((@"%s (%d) : " fmt), __PRETTY_FUNCTION__, __LINE__, ##__VA_ARGS__)
 #else
 #define WY_LOG(...)
 #endif
 
-#define WY_IS_IOS_EQUAL_TO(v)                  YES //([[[UIDevice currentDevice] systemVersion] compare:v options:NSNumericSearch] == NSOrderedSame)
-
-#define WY_IS_IOS_GREATER_THAN(v)              YES //([[[UIDevice currentDevice] systemVersion] compare:v options:NSNumericSearch] == NSOrderedDescending)
-
-#define WY_IS_IOS_GREATER_THAN_OR_EQUAL_TO(v)  YES //([[[UIDevice currentDevice] systemVersion] compare:v options:NSNumericSearch] != NSOrderedAscending)
-
-#define WY_IS_IOS_LESS_THAN(v)                 NO // ([[[UIDevice currentDevice] systemVersion] compare:v options:NSNumericSearch] == NSOrderedAscending)
+#ifdef WINOBJC
+#define WY_IS_IOS_EQUAL_TO(v)                  YES
+#define WY_IS_IOS_GREATER_THAN(v)              YES
+#define WY_IS_IOS_GREATER_THAN_OR_EQUAL_TO(v)  YES
+#define WY_IS_IOS_LESS_THAN(v)                 NO
```

Comment:
@msft-Jeyaram this should be clearer now with the latest pull (also might be easier to review the change in my WYPopoverController fork). The above was an overzealous workaround for an issue accessing UIDevice methods at + (void)load; time. 

Here's a stacktrace for the issue:
```
 	KernelBase.dll!_RaiseException@16()	Unknown
 	vcruntime140d_app.dll!__CxxThrowException@8()	Unknown
 	LIBOBJC2.DLL!objc_exception_throw(void * object) Line 127	C++
 	OBJCUWP.DLL!wil::details::ReportFailure(void * callerReturnAddress, unsigned int lineNumber, const char * fileName, const char * functionName, const char * code, void * returnAddress, wil::FailureType type, HRESULT hr, const wchar_t * message, details::ReportFailureOptions options) Line 3134	Objective-C++
 	OBJCUWP.DLL!wil::details::ReportFailure_Msg() Line 3145	Objective-C++
 	OBJCUWP.DLL!wil::details::in1diag5::Objc_Throw_HrMsg(void * callerReturnAddress, unsigned int lineNumber, const char * fileName, const char * functionName, const char * code, HRESULT hr, const char * formatString, ...) Line 4728	Objective-C++
 	OBJCUWP.DLL!WAIPackageStatics_inst() Line 509	Objective-C++
 	OBJCUWP.DLL!+[WAPackage current](objc_class * self, objc_selector * _cmd) Line 516	Objective-C++
>	UIKIT.DLL!-[UIDevice _newIdentifierForVendor](UIDevice * self, objc_selector * _cmd) Line 71	Objective-C++
 	UIKIT.DLL!-[UIDevice init](UIDevice * self, objc_selector * _cmd) Line 65	Objective-C++
 	FOUNDATION.DLL!+[NSObject new](id * self, objc_selector * _cmd) Line 68	Objective-C++
 	UIKIT.DLL!+[UIDevice currentDevice](objc_class * self, objc_selector * _cmd) Line 50	Objective-C++
 	UIKIT.DLL!+[_WYPopoverTheme theme](objc_class * const self, objc_selector * _cmd) Line 450	Objective-C
 	UIKIT.DLL!+[_WYPopoverController load](objc_class * const self, objc_selector * _cmd) Line 1653	Objective-C
 	LIBOBJC2.DLL!objc_send_load_message(objc_class * class) Line 58	C
 	LIBOBJC2.DLL!objc_resolve_class(objc_class * cls) Line 249	C
 	LIBOBJC2.DLL!objc_resolve_class_links() Line 269	C
 	LIBOBJC2.DLL!__objc_resolve_module(objc_module_abi_8 * module) Line 133	C
 	LIBOBJC2.DLL!__objc_resolve_legacy_modules() Line 182	C
 	UIKIT.DLL!___objc_legacy_resolve_all() Line 84	C++
 	[External Code]	
```